packages/lib-utils/src/k8s/k8s-utils.ts

```diff
@@ -43,10 +65,16 @@ export const getK8sResourceURL = (
   const { ns, name, path, queryParams } = queryOptions;
   let resourcePath = getK8sAPIPath(model);
 
+  const activeWorkspace = getActiveWorkspace();
+  if (activeWorkspace) {
+    resourcePath += `/${activeWorkspace}`;
```

Comment:
@BlakeHolifield this is duplicated with what we already do in `getK8sAPIPath` above